src/index.js

```diff
@@ -39,9 +39,10 @@ class DragSortableList extends React.Component {
 
   componentDidMount() {
     const draggableChildrenSelector = '#' + this.ref + '> .draggable'
+    const noDragEle = event.target.querySelector('.no-drag');
     const ignoreNoDrag = fun => event => {
       const mouseElement = document.elementFromPoint(event.clientX, event.clientY)
-      if(mouseElement && !mouseElement.classList.contains('no-drag') && mouseElement.parentNode && !mouseElement.parentNode.classList.contains('no-drag')) {
+      if(!noDragEle || (noDragEle&&!noDragEle.contains(mouseElement))) {
```

Comment:
I think
````
if(!noDragEle || !noDragEle.contains(mouseElement)) {
````
would be enough ?

Reply:
we were checking the parent for the class as well before, you don't seem to do it anymore. Does that mean it was unnecesary ? (it's very possible ^^)

Reply:
no , I suggest you think about mine idea，It better check the drag experience

Reply:
"if(!noDragEle || !noDragEle.contains(mouseElement)) {"
 is better

Reply:
"if(!noDragEle || !noDragEle.contains(mouseElement)) {"
is better